REPL flaky test failing

[test_set_timeout_interlaced](https://github.com/denoland/deno/blob/6f79ad721a9f8c9d66d79f21ea479286f3ca5374/tools/repl_test.py#L91) test is failing for me, but doubling `setTimeout` and `sleep` values fixes it.

----------

More details:
I'm using PowerShell on Windows 10 and the [latest](https://github.com/denoland/deno/tree/cae71ed8416cba63a2be8c9068b3a3e0b148a32a) master branch.
Issue persists when running tests on both debug and release build, but in an inconsistent way.
I have an 8th gen i7 CPU and 16GB of RAM, so my laptop should perform just fine and I don't think that it's a performance issue.

When I run the test as is:
```
"setTimeout(() => console.log('a'), 500)",
"setTimeout(() => console.log('b'), 300)",
sleep=0.4)
```
with **debug** and **release** builds, it fails with the following message:
```
repl_test.py
......Traceback (most recent call last):
  File "tools\test.py", line 79, in <module>
    sys.exit(main(sys.argv))
  File "tools\test.py", line 71, in main
    repl_tests(deno_exe)
  File "C:\Users\MM660\WebstormProjects\deno4\deno\tools\repl_test.py", line 125, in repl_tests
    Repl(deno_exe).run()
  File "C:\Users\MM660\WebstormProjects\deno4\deno\tools\repl_test.py", line 47, in run
    self.__getattribute__(t)()
  File "C:\Users\MM660\WebstormProjects\deno4\deno\tools\repl_test.py", line 96, in test_set_timeout_interlaced
    assertEqual(out, '1\n2\na\nb\n')
  File "C:\Users\MM660\WebstormProjects\deno4\deno\tools\repl_test.py", line 121, in assertEqual
    raise AssertionError("{} != {}".format(repr(left), repr(right)))
AssertionError: '1\n2\n' != '1\n2\na\nb\n'
```
So, only `1` and `2` are outputted, no `a` and `b`.

Most times, on **release** build, it produces very interesting output:
```
AssertionError: '1\n2\nb\na\n' != '1\n2\na\nb\n'
```
So, the order of output is messed up, `a` and `b` are changing places.

Rarely (about 1/10 times) it gives that on **release** build:
```
AssertionError: '1\n2\nb\n' != '1\n2\na\nb\n'
```
So, only `1`, `2` and `b` are outputted, no `a`.

When I tried to slightly increase the numbers, as following:
```
"setTimeout(() => console.log('a'),700)",
"setTimeout(() => console.log('b'), 500)",
sleep=0.6)
```
I got following output with **debug** build:
```
AssertionError: '1\n2\nb\n' != '1\n2\na\nb\n'
```
So, only `1`, `2` and `b` are outputted, no `a`.

With the same numbers, but **release** build, it worked just fine and tests passed.

----------

So, the point is: this test is really flaky and I'm expecting it to be an issue not only for me.

I will be making PR shortly, where I will double values once again, as @ry did in #1252 previously, as was planned in https://github.com/denoland/deno/pull/1237#issuecomment-442930575.

Let me know is there anything that I'm missing, or if there's a better way to solve that.
Somehow, it's passing on AppVeyor server, so it may be environment or performance issue.
Maybe I just have wrong environment or worse performance then AppVeyor server.

Also, feel free to up-vote if you are experiencing same issue.

--- a/tools/repl_test.py
+++ b/tools/repl_test.py
@@ -90,9 +90,9 @@ class Repl(object):
 
     def test_set_timeout_interlaced(self):
         out, err, code = self.input(
-            "setTimeout(() => console.log('a'), 500)",
-            "setTimeout(() => console.log('b'), 300)",
-            sleep=0.4)
+            "setTimeout(() => console.log('a'), 1000)",
+            "setTimeout(() => console.log('b'), 600)",
+            sleep=0.8)
         assertEqual(out, '1\n2\na\nb\n')
         assertEqual(err, '')
         assertEqual(code, 0)

